test/govspeak_table_with_headers_test.rb

```diff
@@ -131,6 +131,39 @@ def expected_outcome_for_table_with_blank_table_headers
 )
   end
 
+  def expected_outcome_for_table_headers_containing_links
+    %(
+<table>
+  <thead>
+    <tr>
+      <td></td>
+      <th scope="col">Second Column</th>
+      <th scope="col">Third Column</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th scope="row">Link contained in header <a rel="external" href="http://google.com">link1</a>
+</th>
+      <td>Cell</td>
+      <td>Cell</td>
+    </tr>
+    <tr>
+      <th scope="row">No link</th>
+      <td>Cell</td>
+      <td>Cell</td>
+    </tr>
+    <tr>
+      <th scope="row"><a rel="external" href="http://www.bbc.co.uk">Whole header is link</a>
+</th>
+      <td>Cell</td>
+      <td>Cell</td>
+    </tr>
+  </tbody>
+</table>
+)
+  end
```

Comment:
This might be overkill so please guide me, but for this test and the one below using govspeak syntax should there be some assertions to ensure that the link is rendering? I'm not familiar with govspeak's testing stack so as I say, this might be too much and I might just be being daft.

Reply:
I think you're missing what's happening. This is the expected output and the govspeak syntax is the input. The actual test is further down and compares the two.

Reply:
🙈 Oops. Comment withdrawn.